src/panels/lovelace/components/hui-image.ts

```diff
@@ -242,7 +320,7 @@ export class HuiImage extends LitElement {
     // an order of magnitude smaller.
     const width = Math.ceil(element_width * SCALING_FACTOR);
     // If the image has not rendered yet we may have a zero height
-    const height = this._image.offsetHeight
+    const height = this._image?.offsetHeight
```

Comment:
Same

Reply:
Couldn't do it here because we need to do math on both sides of the ternary..unless I missed something?

Reply:
Right. We could prevent the double call to offset height by putting that in a const, but not really needed 🤷🏻‍♂️

Reply:
Cant we use `_lastImageHeight` here btw when available?

Reply:
I think so

Reply:
I used that.  

I tried to use the `div` for the first render but it ended up that it renders and then reflows so the first height is too small so you end up with a blurry image.